catalog/app/components/Preview/loaders/Tabular.tsx

```diff
@@ -0,0 +1,191 @@
+import * as R from 'ramda'
+import * as React from 'react'
+
+import { HTTPError } from 'utils/APIConnector'
+import * as AWS from 'utils/AWS'
+import * as Config from 'utils/Config'
+import * as Data from 'utils/Data'
+import { mkSearch } from 'utils/NamedRoutes'
```

Comment:
this function can now be imported directly from [`utils/mkSearch`](https://github.com/quiltdata/quilt/blob/master/catalog/app/utils/mkSearch.ts)